scripts/create-api.py

```diff
@@ -13,10 +14,29 @@
 apps = os.listdir(ANNOTATIONS_DIR)
 
 
+def linkify(text):
+    """
+    Gives bare URLs and additional context
+
+    This saves effort when writing annotations. Currently only handles
+    STMO URLs
+    """
+    for stmo_pattern in [
+        "(https://sql.telemetry.mozilla.org/queries/([0-9]+)[^\s]*)",
+        "(https://sql.telemetry.mozilla.org/dashboard/([A-z\-]+)[^\s]*)",
```

Comment:
Slight preference for these to be named constants defined once outside of this function like `STMO_QUERIES_RE = re.compile(...)`. This would save compiling the re on every use and I'd argue make it more readable (`text = stmo_pattern.sub(r"[STMO#\2](\1)🔒", text)`)